README.md

```diff
@@ -65,6 +65,45 @@ app('api.router')->version(config('api.version'), ['namespace' => 'Saritasa\Lara
   });
 ```  
 
+```php
+<?php
+
+class LoginRequest extends FormRequest implements ILoginRequest
+{
+    /**
+     * Allow all users access here.
+     * 
+     * @return bool
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function rules(): array
+    {
+        return [
+            'email' => 'required|string',
+            'password' => 'required|string',
+        ];
+    }
+}
+
+```
+
+```php
+class AuthApiController extends JWTAuthApiController
```

Comment:
Api controller will not change, actually. but you will have to add custom binding in application - show it here, in documentation.
Actually, this docs should match to changes which you make in your project - a working example.

Reply:
done